test/SE-13092.test.js

```diff
@@ -44,8 +44,8 @@ describe('SE-13092', () => {
         assert.typeOf(result, 'array');
         const item = result[0];
         const { value } = item;
-        assert.include(value, '<Person>', 'generates type name (opening tag)');
```

Comment:
Obviously this is wrong. The result of calling some function was to generate a type that has "<Person>" tag in the generated XML. Not it hasn't. This is obviously AMF's issue but if you remove this from the test we will lose track of this issue and our clients again remind us that there's something wrong with examples.

Reply:
To be checked with AMF team

Reply:
This is correct then, giving our discussion